test_pii_detect.py

```diff
@@ -50,6 +50,11 @@ def thats_on_1891(self):
         # merge errors are a common reason for failures
         result_list = show_aggie_pride()
         self.assertEqual(result_list[4], 'Thats on 1891!')
+    def whataggiesdo(self):
```

Comment:
There should be a space between the previous function and your new test function

Your test function must begin with the word test or it will not execute as a test case
test_whataggiesdo() would work